content/syllabuses/data-sci.md

```diff
@@ -9,29 +9,29 @@ _db_id: 2
 - {{% contentlink path="topics/solo-learn/python/module-5-more-types" %}}
 - {{% contentlink path="projects/katas/level-2" flavour="python" %}}
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python" %}}
-- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
 - {{% contentlink path="workshops/general-coding-best-practices" %}}
-- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
-- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
+- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/survey-design" %}}
 - {{% contentlink path="topics/google-forms-like-a-boss" %}}
 - {{% contentlink path="projects/data-science-specific/cross-validation-and-simple-linear-regression/" flavour="python" %}}
 - {{% contentlink path="projects/morse-code" flavour="python" %}}
```

Comment:
Should we get rid of morse code? Or put it later on?

Reply:
Yeah we can remove it, make room for the other data science projects.